Updates for Astro Runtime Multiple Architecture Support (#1481)
astronomer/docs: astro/runtime-image-architecture.md

> ## System distribution

Reviewer: In the distribution section above, we should mention that Runtime image tags are [multi-arch](https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/). They will automatically assume the right CPU architecture based on your current machine.

The following table lists the Debian versions and architectures supported by each Astro Runtime version.

| Astro Runtime | Debian                 | Architecture    |
| ------------- | ---------------------- | -------------   |
| 4.1.x         | 11.3 (bullseye)        | AMD64           |
| 4.2.x         | 11.3 (bullseye)        | AMD64           |
| 5.0.x         | 11.3 (bullseye)        | AMD64           |

> | 6.0.x         | 11.3 (bullseye)        | AMD64           |

Reviewer: ```suggestion
| 6.0.0 - 6.0.3         | 11.3 (bullseye)        | AMD64           |
```

> | 6.0.4         | 11.3 (bullseye)        | AMD64 and ARM64 |

Reviewer: ```suggestion
| 6.0.4 - 6.0.x         | 11.3 (bullseye)        | AMD64 and ARM64 |
```

> :::info
> 
> Astro Runtime 6.0.4 and later support AMD64 and ARM64 processor architectures for local development. The image tag is multi-arch and the correct distribution for the processor architecture on which you're running the image is used by default.
> 
> :::

Reviewer: ```suggestion
Astro Runtime 6.0.4 and later support AMD64 and ARM64 processor architectures for local development. The image tag is multi-arch and the correct distribution for the processor architecture on which you're running the image is used automatically.
```
Because you can't chose which architecture is used, I would call this automatic behavior instead of default behavior.
